packages/ckeditor5-dev-changelog/src/index.ts

```diff
@@ -0,0 +1,6 @@
+/**
+ * @license Copyright (c) 2003-2025, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+export { generateChangelog } from './generateChangelog';
```

Comment:
Linter error:

```
6:1  error  Missing file extension in import/export "./generateChangelog"  ckeditor5-rules/require-file-extensions-in-imports
```

Reply:
Fixed in [725a690](https://github.com/ckeditor/ckeditor5-dev/pull/1084/commits/725a69016251ba9fccf07724db34e5a06a75321c).